mappings/net/minecraft/block/Block.mapping

```diff
@@ -52,6 +52,9 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_9503 getBlockFromItem (Lnet/minecraft/class_1792;)Lnet/minecraft/class_2248;
 		ARG 0 item
 	METHOD method_9504 rainTick (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
+		ARG 1 state
```

Comment:
This has been covered by atleast 3 other PRs. I would probably leave these params.